linux_os/guide/system/bootloader-grub2/grub2_uefi_password/rule.yml

```diff
@@ -8,28 +8,15 @@ description: |-
     The grub2 boot loader should have a superuser account and password
     protection enabled to protect boot-time settings.
     <br /><br />
-    To do so, select a superuser account name and password and and modify the
-    <tt>/etc/grub.d/01_users</tt> configuration file with the new account name.
-    <br /><br />
     Since plaintext passwords are a security risk, generate a hash for the pasword
     by running the following command:
     <pre>$ grub2-setpassword</pre>
     When prompted, enter the password that was selected.
     <br /><br />
-    NOTE: It is recommended not to use common administrator account names like root,
-    admin, or administrator for the grub2 superuser account.
-    <br /><br />
-    Change the superuser to a different username (The default is 'root').
-    <pre>$ sed -i s/root/bootuser/g /etc/grub.d/01_users</pre>
-    <br /><br />
-    To meet FISMA Moderate, the bootloader superuser account and password MUST
-    differ from the root account and password.
-    Once the superuser account and password have been added,
```

Comment:
The removed text still seems valid. What has changed?

Reply:
Partly the description has been moved to the new rule.

I took out those FISMA lines, because profile reference rules and not vice versa - if a rule is required by a security policy, it is selected by the appropriate profile. Generally, we want every rule should fit seamlessly to any profile, and mentioning a policy in rule's description is exactly a seam.
The grub superuser password != system root password requirement could become a new rule, but there is no way to check this by OVAL, so is it even worth it to have in the guide just for the description? And if so, the description would yet have to be written, which is out of scope of this PR.